FluentTerminal.App.ViewModels/MainViewModel.cs

```diff
@@ -214,6 +218,34 @@ public Task AddConfigurableTerminal()
             });
         }
 
+        public Task AddRemoteTerminal()
+        {
+            return ApplicationView.RunOnDispatcherThread(async () =>
+            {
+                var connectionInfo = await _dialogService.ShowSshConnectionInfoDialogAsync();
+
+                if (connectionInfo == null)
+                {
+                    if (Terminals.Count == 0)
+                    {
+                        await ApplicationView.TryClose();
+                    }
+
+                    return;
+                }
+
+                var profile = new ShellProfile
```

Comment:
Eventually we probably want to define a dedicated `RemoteProfile` type but this will probably do for now

Reply:
Yes, that's just a temp solution for now. I think we will need to extend this once we allow to also store connections